### DISCUSSION TITLE
Last date to submit project Synopsis batch 1

### DISCUSSION THREAD

**Discussion by Manish814142** [github/india#587] [Students]
I am from batch 1 and our End date to submit project synopsis is 25th november so my question is, we have 25th also or till 24th november to submit project synopsis???
![Bathc 1](https://user-images.githubusercontent.com/92366137/143063210-0c8bb79e-e214-45fb-9da9-fc8d5cc7d040.PNG)

**Comment by punitjain03** [comment]
Hi Manish, Would be great if you could help me what is this all about? Where you got this details?

**Reply by Manish814142** [reply]
go to project tab then there is option to read about project option download that file and you will get these information's.

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official GitHub Externship Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Link to join: https://discord.gg/ceMXzhfaka
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏
